scripts/west_commands/completion/west-completion.bash

```diff
@@ -802,6 +802,8 @@ __comp_west_runner_cmd()
 	"
 
 	local file_opts="
+		--file
+		--file-type
```

Comment:
Should we include the short forms here as well?

Reply:
Yes, good point. @MeisterBob could you add those please?